prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1358,6 +1360,7 @@ int rxr_ep_init(struct rxr_ep *ep)
 		RXR_PKT_FROM_EFA_TX_POOL,
 		rxr_get_tx_pool_chunk_cnt(ep),
 		rxr_get_tx_pool_chunk_cnt(ep), /* max count */
+		RXR_BUF_POOL_ALIGNMENT,
```

Comment:
Is this change intended for this PR? Does it break anything if omitted?

Reply:
I added an alignment argument to this function, all the pkt pool is still using the same `RXR_BUF_POOL_ALIGNMENT` alignment except for the readcopy pkt  pool, which will use 128 alignment. See the commit message in https://github.com/ofiwg/libfabric/pull/8841/commits/551a9e0d08a9889d1e9919e80d36d3e1a2be00cc